crypto/x509/v3_ind_iss.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 1999-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include "internal/cryptlib.h"
+#include <openssl/asn1.h>
+#include <openssl/asn1t.h>
+#include <openssl/x509v3.h>
+#include "ext_dat.h"
+
```

Comment:
Could you add the reference as a comment in each of the extension files..
e.g
````
/*
 * See https://...... Section 7.5.2.5 Indirect issuer extension
 */
````